tests/scripts/run-xunit-perf.cmd

```diff
@@ -2,193 +2,297 @@
-shift
-shift
-goto :ARGLOOP
-)
-IF /I [%1] == [-testEnv] (
-set TEST_ENV=%2
-shift
-shift
-goto :ARGLOOP
-)
-IF /I [%1] == [-configuration] (
-set TEST_CONFIG=%2
-shift
-shift
-goto :ARGLOOP
-)
-if /I [%1] == [-?] (
-goto :USAGE
-)
-if /I [%1] == [-help] (
-goto :USAGE
-)
-
-if [%CORECLR_PERF%] == [] (
-goto :USAGE
-)
-
-goto :SETUP
+    type "%XMLOUT%" | findstr /i /c:"test name"
+  )
+
+  rem Save off the results to the root directory for recovery later in Jenkins
+  call :run_cmd xcopy "Perf-%BENCHNAME%*.xml" "%CORECLR_REPO%\" || exit /b 1
+  call :run_cmd xcopy "Perf-%BENCHNAME%*.etl" "%CORECLR_REPO%\" || exit /b 1
+
+  exit /b 0
+
+:parse_command_line_arguments
+rem ****************************************************************************
+rem   Parses the script's command line arguments.
+rem ****************************************************************************
+  IF /I [%~1] == [-testBinLoc] (
+    set CORECLR_PERF=%CORECLR_REPO%\%~2
+    shift
+    shift
+    goto :parse_command_line_arguments
+  )
+  IF /I [%~1] == [-scenarioTest] (
+    set IS_SCENARIO_TEST=1
+    shift
+    goto :parse_command_line_arguments
+  )
+  IF /I [%~1] == [-runtype] (
+    set BENCHVIEW_RUN_TYPE=%~2
+    shift
+    shift
+    goto :parse_command_line_arguments
+  )
+  IF /I [%~1] == [-library] (
+    set TEST_FILE_EXT=dll
+    shift
+    goto :parse_command_line_arguments
+  )
+  IF /I [%~1] == [-uploadtobenchview] (
+    set BENCHVIEW_PATH=%~2
+    shift
+    shift
+    goto :parse_command_line_arguments
+  )
+  IF /I [%~1] == [-arch] (
+    set TEST_ARCHITECTURE=%~2
+    shift
+    shift
+    goto :parse_command_line_arguments
+  )
+  IF /I [%~1] == [-testEnv] (
+    set TEST_ENV=%~2
+    shift
+    shift
+    goto :parse_command_line_arguments
+  )
+  IF /I [%~1] == [-configuration] (
+    set TEST_CONFIG=%~2
+    shift
+    shift
+    goto :parse_command_line_arguments
+  )
+
+  if /I [%~1] == [-?] (
+    call :USAGE
+    exit /b 0
+  )
+  if /I [%~1] == [-help] (
+    call :USAGE
+    exit /b 0
+  )
+  if [%CORECLR_PERF%] == [] (
+    call :USAGE
+  )
+
+  exit /b %ERRORLEVEL%
+
+:set_test_architecture
+rem ****************************************************************************
+rem   Sets the test architecture.
+rem ****************************************************************************
+  IF /I [%TEST_ARCHITECTURE%] == [x86jit32] (
+      set TEST_ARCH=x86
+  ) ELSE (
+      set TEST_ARCH=%TEST_ARCHITECTURE%
+  )
+  exit /b 0
+
+:verify_core_overlay
+rem ****************************************************************************
+rem   Verify that the Core_Root folder exist.
+rem ****************************************************************************
+  set CORECLR_OVERLAY=%CORECLR_REPO%\bin\tests\Windows_NT.%TEST_ARCH%.%TEST_CONFIG%\Tests\Core_Root
+  if NOT EXIST "%CORECLR_OVERLAY%" (
+    call :print_error Can't find test overlay directory '%CORECLR_OVERLAY%'. Please build and run Release CoreCLR tests.
```

Comment:
It looks like the apostrophe in can't is treating everything like its quoted out. I don't know if that's just a github thing though